vespa-athenz/src/main/java/com/yahoo/vespa/athenz/client/zts/ZtsClient.java

```diff
@@ -103,5 +106,15 @@ X509Certificate getRoleCertificate(AthenzRole role,
                                        KeyPair keyPair,
                                        String cloud);
 
+    /**
+     * For a given provider, get a list of tenant domains that the user is a member of
+     *
+     * @param providerIdentity Provider identity
```

Comment:
This whole javadoc section can be removed, as it doesn't add any more context than the parameter and method name.

Reply:
Agree. I did include them because of the excessive number of Javadoc warnings that was outputted by Maven some time ago ;)